includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -521,7 +519,7 @@ private function wc_setup_redirect_to_wc_admin_onboarding() {
 		$onboarding_data = get_option( 'wc_onboarding_profile', array() );
 		update_option( 'wc_onboarding_profile', array_merge( $onboarding_data, $profile_updates ) );
 
-		wp_safe_redirect( esc_url_raw( admin_url( 'admin.php?page=wc-admin' ) ) );
+		wp_safe_redirect( wc_admin_url( '&enable_onboarding=1' ) );
```

Comment:
I realize the only current code path to this method already has a check to ensure `wc_admin_url` exists, but should we add one here now too just to be extra safe?

Reply:
This redirect should also have an `exit;`

Reply:
There's actually an `exit` above right after this is called, but moved it down here to avoid any confusion.

Reply:
>I realize the only current code path to this method already has a check to ensure wc_admin_url exists, but should we add one here now too just to be extra safe?

Added in https://github.com/woocommerce/woocommerce/pull/25527/commits/236bae9b6a8ed06f638e66cb0e2e4ba8b11fbd11